server/models/Collection.test.js

```diff
@@ -416,3 +418,63 @@ describe("#membershipUserIds", () => {
     expect(membershipUserIds.length).toBe(6);
   });
 });
+
+describe("#findByPk", () => {
+  test("should return collection with collection Id", async () => {
+    const { collection } = await seed();
```

Comment:
```suggestion
    const collection = await buildCollection();
```

`seed` is quite slow as it builds a bunch of different models, I'm trying to avoid using it in new tests in favor of individual factories like above

Reply:
Oh! For some reason, I thought seed should be fast 😅